api_v3/services/UserService.php

```diff
@@ -45,9 +45,10 @@ function addAction(KalturaUser $user)
 		return kLock::runLocked($lockKey, array($this, 'adduserImpl'), array($user));
 	}
 	
-	function addUserImpl($user)
+	function addUserImpl(KalturaUser $user)
 	{
 		$dbUser = null;
+		$user->validateForInsert();
```

Comment:
I think you should replace these two lines (validateForInsert and toObject) with 
$user->toInsertableObject